client/dist/html/pages/tom_clancy.html

```diff
@@ -1,4 +1,5 @@
 <div id="tom_clancy">
+	<audio autoplay="autoplay" loop="loop"><source src="wup.mp3" type="audio/mpeg"/></audio>
```

Comment:
I see - there's no JS/CSS file so it keeps the hashes the same. This is a great improvement - the only problem is, it doesn't work.

👎